src/hermite_multidimensional.hpp

```diff
@@ -52,10 +74,20 @@ std::vector<int> find_rep(int val, int base, int n) {
     }
 
     return digits;
-
 }
 
 
+/**
+ * Renormalizes an unnormalized photon number statistics of a Gaussian state.
+ * Based on the MATLAB code available at: https://github.com/clementsw/gaussian-optics
+ *
+ * @param tn unnormalized flattened vector of size \f$res**nmodes$ representing unnormalized photon number statistics
+ *       \f$2n\times 2n\f$ row-ordered symmetric matrix.
+ * @param nmodes number of modes
+ * @param res highest number of photons to be resolved.
+ *
+ * @return Renormalized photon number statistics
+ */
```

Comment:
@bgupt, this one needs to be added as well for the sqrtfactorial function (currently it is just a copy-paste of the one below)

Reply:
Done